src/schema/schema.ts

```diff
@@ -190,8 +190,14 @@ export function createZimbraSchema(
 						}))
 			},
 			Mutation: {
-				action: (_, variables) => {
+				action: (_, variables, context) => {
```

Comment:
please pass a default value for context as `context = {}`, might fail when we try to access the `local` from context.